### ISSUE TITLE
Docker version not starting

### ISSUE THREAD

**Issue by neocox** [peachdocs/peach#27]
VERSIONS:

```
$ docker --version
Docker version 1.11.2, build b9f10c9
$ docker exec peach /app/peach/peach --version
Peach version 0.9.2.1214
```

I'm following the docker instructions at:
https://github.com/peachdocs/peach/tree/master/docker

But when I run the following command it hangs and never start:

```
$ docker run --name=peach -p 5555:5555 -v peach-data:/data/peach peachdocs/peach
Jun 20 14:34:08 syslogd started: BusyBox v1.24.1
```

Some aditional info:

```
$ docker exec peach ps
PID   USER     TIME   COMMAND
    1 root       0:00 /bin/s6-svscan /app/peach/docker/s6/
   15 root       0:00 s6-supervise syslogd
   16 root       0:00 /sbin/syslogd -nS -O-
   20 root       0:00 ps

$ docker exec -it peach netstat -an
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address           Foreign Address         State
Active UNIX domain sockets (servers and established)
Proto RefCnt Flags       Type       State         I-Node Path
unix  2      [ ]         DGRAM                    29186355 /dev/log

$ docker logs peach
Jun 20 14:35:28 syslogd started: BusyBox v1.24.1
```

**Comment by Unknwon** [comment]
I have network difficulties to pull Docker images, will try few days later.

**Comment by amosipov** [comment]
I have the same issue. Any updates on this? Thanks in advance.

**Comment by amosipov** [comment]
Looks like `peach` folder is missing in `peach/docker/s6/`

**Comment by Unknwon** [comment]
Peach is not even started inside the container... I'm running out of my knowledge about Docker...

**Comment by jbpratt78** [comment]
Hi, :smile: can this be closed as per #41 was merged and building with Docker works?
